### ISSUE TITLE
pro plugins: please clearify strategy (via blog post)

### ISSUE THREAD

**Issue by hpvd** [piwik/piwik#9169]
The Pro Plugins on 
https://piwik.pro/plugins/
look really promising.
And number of pro plugins is rising.

If I get everything right, at this time these plugins only be available 
1) within the piwik cloud or
2) together with some kind of service contract

So the question is:
**what's the aim of this and what for future plans exist for great PiwikPro Plugins,
to make them accessible keeping the main advantages of Piwik?**
[the ownership of data and an easy usable and affordable platform]

-------------------------------------------------------------------------------------------------------------------------------------------
Background:
to 1)
Piwik in a cloud undermine imho one of the biggest advantages: the total owner ship of all data.
This is not only important regarding trust but also regarding possibilities of website setup etc (only own site(s) serve all data).
"Not selfhosting" lessen the advantages to free GA.

to 2)
the obligation of service contracts dramatically reduce the number of people who use something just because not everybody likes service contracts...not only because of costs...

**Comment by hpvd** [comment]
=> just to clarify this post: I'm not against the possibility to earn some money (which is of course needed!),
but if there are great ideas, technics and plugins, imho one should give as many people as possible easy access to them - not imperative for free but in an uncomplicated, fast and affordable way and with keeping the biggest advantage of Piwik alive.

**Comment by hpvd** [comment]
Just to look how other do this:
e.g. Magento: there are 10000 of extensions available for a realistic amount of money
(typ. betwenn 19-199USD)
directly for download and usage on your own infrastructure - within 5minutes.

**Comment by mattab** [comment]
Hi @hpvd 

Here is more information:

The premium features (premium plugins) are part of Piwik Enterprise and Piwik Cloud. Every client of Piwik Cloud and Piwik Enterprise gets access to all premium plugins at once. There is currently no way to buy only one particular plugin. 

This is quite similar model to [Magento Enterprise edition](http://magento.com/products/enterprise-edition/marketing-and-merchandising). 

In the future we may consider selling the plugins on the Marketplace separately (for a lower price) but this is not planned for now, as we have decided to include all plugins for all our customers.

**Comment by tsteur** [comment]
I'm pretty sure during the meetup it was discussed to release a blog post and maybe a page explaining the role Piwik vs Piwik PRO etc. Should we create a new issue for this or move this issue maybe in current milestone? Or will someone from Piwik PRO work on that blog post?

**Comment by mattab** [comment]
@tsteur Thanks for the reminder, I'm putting this issue in 2.15.1 so we will write a blog post about this topic on Piwik Blog in coming weeks.

**Comment by masteranalyze** [comment]
"Just to look how others do this:
e.g. Magento: there are 10000 of extensions available for free or for a realistic amount of money
(typ. between 19-199USD)
directly for download and ready for enhancing your existing Magento installation on your own infrastructure - pay, download, install, use - all within 5minutes"

I agree with hpvd,i think that all plugins should be offered in the cloud edition all togheter and piwik will have:

a)Revenue stream from Cloud based platform
b)Another revenue stream from selling the plugins and let others sell their plugins or additionaly features for piwik,in an marketplace ,this will generate more revenue for piwik on the long term.

Let`s think about it,as financial liberation too,not only analytics.

The people that are going for the cloud edition,are not the same one that will pay you 50$ for an plugin.
More people can get into community and helps us do an more good product,if core team members finds an way through an marketplace to give them some of the revenue,that way everybody haves to win,and on the long run,piwik haves the most of the win,both financially and both on developing.

People that just want an plugin,let`s face it,they will never buy the cloud edition,what does that mean for piwik ?It means maybe 10,20,30,50$ bucks losed.

Why ?When we can be flexible and generate revenue from both sides.

Like hpvd said,magento is very succesfully and it`s succes is based on liberation and by providing in parallel cloud based pro edition and also marketplace for those who wish to extend the magento functionality.

I think is an great idea,that can generate unlimited earning potential,and piwik core team members should defintely think on liberating plugins also.

And from my data as i know one guy working in magento,they generate more money from their marketplace then from their pro edition.

The basic pro piwik edition to offer all your plugins into cloud,does not interfere at all with offering to somebody an plugin for an lower fee.

It will just add another revenue stream for piwik,witch i think is an great thing,as more developers and people join in,and helps,that s the most good thing that can happen,if we really want piwik to be the best analytics from the market,and not to be only analytics for the web,but also for mobile app,for IoT ,that`s the future!

**Comment by hpvd** [comment]
just to share three ideas from lessons learned working with magento
(because it's not done that way there):

Marketplace is the perfect place for developers to present, advertise and sell there Plugins to the crowd.

With this "power" one can give 3 simple rules for all or at least all non-free plugins
which **helps a lot to have an perfect working and secure ecosystem with happy users**:

1. Plugins presented in marketplace has always to be compatible to latest version of Piwik
(to give developers a little time e.g. there is a final check 6 weeks after a new release
-> non compatible plugins are  disabled at this day)

2. Plugins presented in market place has to be sold with free live time up dates for buyers:
as long as the plugin in market places and with this receives needed updates
all buyer got access to latest version - no matter when they bought it.

3. Usage of ioncube or similar protections are  not allowed

Not having these rules,
- many many Piwik user often would (like magento users already have to) wait for several month until compatibility-updates of all their plugins are available before being able to update Piwik.
- Other Piwik users would not even update because of the costs of having to pay again for updates.
- And if it's encrypted one can't work on a fix when plug in is not maintained any more.

**Comment by masteranalyze** [comment]
+1 for the rules 
+1 for the liberating plugins 
+1 for marketplace creation along lunching piwik 3.

I totaly agree once again with what hpvd write,and he is right about those rules,that needs to be put for the marketplace.

The difference between the PRO or the cloud services and marketplace,is that cloud are mostly for Business ,agencies ,end users too,but mostly of the customers of the CLOUD pro version are Companies or simple users,maybe someone can clear us out ?

As i think most of them are companies,that really understand the benefits of having an software into the cloud with all plugins,supports,etc.

The marketplace,will be for the end user,is like retail,and with retail more developers will join in,if the rules for the marketplace are in place like hpvd wrote.

The retail user,is very differently from the business sector.

The retail market is huge,comparated to just business.

Earlier those days,google had to publish the results of his Android Marketplace,witch only haves 7 years,and in android marketplace we all know it,that you can find there even 0.99$ app,but from those kind of app,google did make 31 billion $ turnover,expenses 10billion$ ,21billion$ net profit,that means  an total of 3billion $/ net profit per year only from Android MARKETPLACE!

That marketplace,bring them alot of revenue,and is an open source market.

Now imagine only if 0.1% from that can come additionally to piwik basic income,that can mean alot of things will be done more faster,more people will join in,the community will grow up,developers will start to work on all kinds of plugins,it practically means Evolution.

Is no need to re-invent the wheel,if we just look on what the bigs guys like magento or google are doing,we can follow up....

**Comment by masteranalyze** [comment]
And even ,do it more better !As piwik is 10 times more good then google analytics,and tells you more,and if others join in,the future of having analytics for Iot,it will become more faster an reality!

Google analytics is an piece of garbage,that only helps GOOGLE,not the webmaster or the owner of the website!

And all the data collected by them,is used against you!How ?!Simple ,in RANKING you in google,i`ve done hard test on myself.

And when i talked to their software engineers from GOOGLE,they did admited that those stats are not accurate at all,at that nobody from google does not guarantee for them,more exactly they are JUST SAMPLE DATA,that is what their software enginners told me,data is not real time,data can be screwed up by alot of things,and google does not take no responsability of the accuracy of the data google analytics is providing,and if you don`t belive me,just call them and ask them by yourself!

That is what is google analytics,sample data,not REAL ANALYTICS!

Piwik is Real Analytics!

**Comment by mattab** [comment]
Hello, we won't communicate directly about Piwik PRO plugins. We will for sure communicate more in the future as we were quiet recently, stay tuned :+1:

**Comment by mattab** [comment]
FYI as you may have noticed we've completely changed our strategy with regards to paid plugins. Now premium plugins can be purchased separately on the Marketplace: http://plugins.piwik.org/premium 

and we are no more related to Piwik PRO: https://piwik.org/faq/new-to-piwik/faq_21984/
